pkg/server/container_status.go

```diff
@@ -41,7 +41,7 @@ func (c *criContainerdService) ContainerStatus(ctx context.Context, r *runtime.C
 	// * ImageSpec in container status is image tag.
 	// * ImageRef in container status is repo digest.
 	spec := container.Metadata.Config.GetImage()
-	imageRef := container.ImageRef
+	imageRef := container.Metadata.ImageRef
```

Comment:
This doesn't matter I think.

Reply:
consistent with "container.Metadata.Config.GetImage()"

Reply:
It is better to remove it.:)